### ISSUE TITLE
Could not get it to work with fresh gmail email

### ISSUE THREAD

**Issue by jotwin** [tasseff/SecretSanta.jl#2]
I opened a gmail account and turned of secure app login. I couldn't get secret santa to work. I checked I have correct server smtp.gmail.com, port 465, username and password. I get this error 

`curl_easy_perform() failed: SSL peer certificate or SSH remote key was not OK`

**Comment by tasseff** [comment]
I actually ran this on macOS recently and encountered the same issue. What operating system are you using? After switching over to my Ubuntu 18.04 LTS machine and running it, it worked.

**Comment by jotwin** [comment]
Catalina

**Comment by tasseff** [comment]
Yes, this was the same operating system I was using when I encountered the error. I think it has to do with what trusted certificates are already present on macOS versus something like Ubuntu. I'm not sure why the Gmail certificate would not be trusted by default, and when I was investigating this issue, I could not find a fast fix for it. I'm also hesitant to suggest using some insecure variant of the function. I will leave this issue open until we discover some remedy.
